backend/src/websockets/socket.ts

```
@@ -0,0 +1,257 @@
+          
+          errorResponse.writeUInt8(CommandType.ERROR, 0);
+          errorResponse.writeUInt8(ErrorCode.AUTH_FAILED, 1);
+          errorResponse.writeUInt16BE(messageBytes.length, 2);
+          messageBytes.copy(errorResponse, 4);
+          
+          this.send(errorResponse);
+        }
+        break;
+
+      case CommandType.CONFIG_REQ:
+        const response = Buffer.alloc(9);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)